Socket/client/chatapp/src/App.css

```diff
@@ -0,0 +1,15 @@
+.font-face-gb {
```

Comment:
Use https://styled-components.com/docs/api#createglobalstyle instead of css files for global styles